apps/src/blockTooltips/tooltipUtils.js

```diff
@@ -0,0 +1,130 @@
+/* global ace */
+'use strict';
+
+/**
+ * @typedef {Object} parameterSlotInfo
+ * @property {string} funcName
+ * @property {number} currentParameterIndex
+ */
+
+var ONLY_WHITESPACE_REGEXP = /^[ ]*$/;
+var ENDING_OF_BLOCK_COMMENT = /^\*\/$/;
```

Comment:
Oops, the ^ is bug intro'd when trying to undo the lodash `endsWIth` inclusion. Fixed. Also found that `/* something */` is treated as a `comment` while `/** something */` is treated as a `comment.doc`, so added handling of that case as well.